audbackend/core/utils.py

```diff
@@ -72,6 +73,10 @@ def check_version(version: str) -> str:
     return version
 
 
+def date_format(date: datetime.datetime) -> str:
+    return date.strftime('%Y/%m/%d')
```

Comment:
I checked in `audmodel` we actually return with `-` instead of `/`:

![image](https://user-images.githubusercontent.com/10383417/236169917-f54dd61c-2202-4a56-bb46-06f92e4f6fda.png)

Reply:
Using `-` or `/` seems just a convention to me. Do you have a preference?

Reply:
According to https://docs.python.org/3/library/datetime.html#datetime.date.strftime `-` seems to be the isoformat:

![image](https://user-images.githubusercontent.com/10383417/236173024-08dcc9d2-d569-4059-9039-34f15cc93c9e.png)

So maybe change to `-` then?

Reply:
There seems to be an ISO standard using `-`: https://en.wikipedia.org/wiki/ISO_8601

Reply:
done